src/openads-appnexus/repository/PullingDataEntry.js

```diff
@@ -0,0 +1,44 @@
+const TIMEOUT_EXCEPTION = 'Timeout retrieving the Ad from the server'
+
+export default class PullingDataEntry {
```

Comment:
Fucking great! nice encapsulation, makes very straighforward the code and very clear the idea of what's doing.
![](https://media.giphy.com/media/ToMjGpnXBTw7vnokxhu/giphy.gif)

Reply:
Thanks :) 
![](https://media.giphy.com/media/hTh9bSbUPWMWk/giphy.gif)